index.html

```diff
@@ -7,10 +7,94 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Responsive Cake webpage</title>
     <!-- Add a link to your css file here -->
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Dancing+Script&display=swap" rel="stylesheet">
+
 </head>
 
 <body>
     <!-- Add your markup here -->
+    <header>
+      <div class="nav-container">
+
+        <div>
+           <img class="nav-img" src="https://media.istockphoto.com/photos/100th-birthday-cake-picture-id1355524553?b=1&k=20&m=1355524553&s=170667a&w=0&h=9h4foTkneAQzZEtoBrKhFoiMXy1H6bL4bLLKqO9-04M=">
+        </div>
+        
+        <a href="#" class="toggle-button">
+            <span class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+        </a>
+
+        <div class="nav-bar"> 
+          <nav> 
+            <ul>
+                <li><a href="#">HOME</a></li>
+                <li><a href="#">CAKES</a></li>
+                <li><a href="#">ORDERING</a></li>
+                <li><a href="#">LESSONS</a></li>
+                <li><a href="#">ABOUT</a></li>
+            </ul>
+          </nav>
+        </div>
+
+        <p>The best cakes in town delivered to your door</p> 
+
+      </div>
+    </header>
+
+     
+
+   <main>
+
+       <div class="hero-info-img">
+          <h1>Welcome</h1>
+          <p>We are family run bakery.We can make any kind of cake for your birthday party or wedding or specail occasions. You can order it by phone or thorugh our online form.</p>
+          <img class="heroImg" src="https://media.istockphoto.com/photos/strawberry-and-cream-sponge-cake-on-white-wooden-table-picture-id1294578109?b=1&k=20&m=1294578109&s=170667a&w=0&h=cK7uKAhs3r08JVUA22omHA-ToHRr5duIRagkoJwBJHc=">
+        </div>
+
+        
+        <!-- <div class="hero-section"> -->
+        <div class="allImages">
```

Comment:
Try to stick to a single convention on class naming (in CSS, usually it's hyphen separation - the 'camel case' you've used here is more of a JavaScript convention).

Here's one common way of structuring your names: http://getbem.com/naming/

Reply:
Thank you Dom. I will make changes accordingly.